### PR TITLE
Add command 7za, unrar and git-svn command

### PR DESCRIPTION
**Pull request by joelhy**
add 7zip command line utility 7za and unrar command support.

### CODE REVIEW

**Comment by joelhy** [comment]
git-svn command added.

File: pages/common/7za.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+`7za a {{compressed.7z}} {{directory_or_file}}`
+
+- decompress an existing 7z file
+
+`7za x {{compressed.7z}}`
+
+- compress to zip format
+
+`7za a  -tzip {{compressed.zip}} {{directory_or_file}}`
```
Comment (ostera) [inline pages/common/7za.md]: there's an unnecessary extra space between the `a` and the `-tzip` here

File: pages/common/7za.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+- create multipart 7zip file
+
+`7za -v{{size}}{{[b|k|m|g]}} {{compressed.7z}} {{directory_or_file}}`
```
Comment (ostera) [inline pages/common/7za.md]: so if I understood correctly, `7xa -v12g a.7z here` should make a multipart zip file with each part having at most 12 gigs, of what, of the directory? or of the 7z file? will the 7z file be split into several smaller files and stored within the directory?

This one is quite unclear. Try something like:

```
- create a multipart 7zip file; part_size being size plus factor (b for byte,k for kilobyte)
7za -v{{part_size}} ....the rest here
```

It's easy when to read when you understand the command, the point is that most people won't, and thus it has to be written for them instead

File: pages/common/unrar.md [hunk 3]
```diff
@@ -0,0 +1,19 @@
+# unrar
+
+> Extract RAR archives
+
+- show help
+
+`unrar`
```
Comment (ostera) [inline pages/common/unrar.md]: unnecessary

**Comment by igorshubovych** [comment]
In future, plz create 3 separate pull requests for 3 different pages.

File: pages/common/7za.md (hunk 1 above)
Comment (joelhy) [inline pages/common/7za.md]: extra space removed

File: pages/common/7za.md (hunk 2 above)
Comment (joelhy) [inline pages/common/7za.md]: some clarifications committed: create multipart 7zip file; part_size: specifies part size in Bytes, Kilobytes, Megabytes or Gigabytes.
may be the comment is too long.

**Comment by joelhy** [comment]
@igorshubovych Thanks for your advice.

File: pages/common/unrar.md (hunk 3 above)
Comment (joelhy) [inline pages/common/unrar.md]: removed

**Comment by joelhy** [comment]
Split into separate pull requests:
1. 7za: https://github.com/tldr-pages/tldr/pull/590
2. git-svn: https://github.com/tldr-pages/tldr/pull/591
3. unrar: https://github.com/tldr-pages/tldr/pull/592

**Pull request closed without merging**